fameal/ape-tax
# feature: Qualitify update

## What it does ✨
As [promised on twitter](https://twitter.com/TBouder/status/1420350117811572740?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1420350117811572740%7Ctwgr%5E%7Ctwcon%5Es1_c10&ref_url=https%3A%2F%2Fpublish.twitter.com%2F%3Fquery%3Dhttps3A2F2Ftwitter.com2FTBouder2Fstatus2F1420350117811572740widget%3DTweet), the Qualitify of Life Update.
There is no specific feature with this PR. It's mainly code cleaning, some optimisation and stuff like that.

- 📄   Updating the readme to handle `stealth` and `endorsed` status + change the name of `config.js` to `vaults.json` and remove unused `BLOCK_ACTIVATED` for now
- 🔥   Removing most of the unused code : unused imported modules, unused variables, unused methods and functions, 
- 🚒   Moving `config.js` to `vaults.json` (as discussed) and fix the integration with the dApp with as little change as possible : the variable name is still `config` and still imported/init at the same spot. Only change on the `VAULT_ABI` which now is a string. This string will be used to load the correct ABI in the `loadVaultABI` function (`/src/drizzleOptions.js - l07`).
- 🔄   Replace the hardcoded variables such as `max_uint`, `BN_ZERO` or `ADDRESS_ZERO` with the ethers constant.
- 🚨   Adding some conditions to avoid warning and error (div by 0, undefined variables etc.). Console only.

## How to test ✅
The project should be able to start with a `yarn && yarn run serve`.
No change should be visible on the UI. Yeah.
